### PR TITLE
LiveTracking: First request to get trip details to show customer, use…

### PR DESCRIPTION
**Pull request by premkumar91**
First get trip details request and placing customer, user and partner marker on map done.

### CODE REVIEW

File: src/route/route.js [hunk 1]
```diff
@@ -19,5 +20,9 @@ export const router = createBrowserRouter([
     {
         path: '/partner/mapMarker',
         element: <MapWithMarker />
+    },
+    {
+        path: '/customer/liveTracking',
```
Comment (SanthoshGugan) [inline src/route/route.js:25]: can we change url to triplivetracking

File: src/component/LiveTracking.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        apiKey: apiKey,
+        version: "weekly"
+      });
+
+    loader.load().then(async () => {
+        const req = {
+            trip_id: 13,
+            partner_id:4 
+        };
+        const tripResponse = await getTripDetails(req);
```
Comment (SanthoshGugan) [inline src/component/LiveTracking.js:24]: we can move the getTripDetails out of load callback as it is not dependent on gmap load. probab;y we can slipt into separate functions :+1:

Comment (premkumar91) [reply src/component/LiveTracking.js:24]: after trip details completion, we will get lat,lng which is supposed to be feed into map but making two thing async will produce race conditions.

**Review by SanthoshGugan — APPROVED** [review]
LGTM!

**Pull request merged**